helpdesk_mgmt_ticket_tree_recent/views/helpdesk_ticket_views.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version='1.0' encoding='utf-8'?>
+<odoo>
+    <record id="helpdesk_ticket_view_tree_write_date" model="ir.ui.view">
+        <field name="name">helpdesk.ticket.view.tree (write_date)</field>
+        <field name="model">helpdesk.ticket</field>
+        <field name="inherit_id" ref="helpdesk_mgmt.ticket_view_tree"/>
+        <field name="arch" type="xml">
+            <xpath expr="//field[@name='create_date']" position="after">
+                <field name="write_date" />
```

Comment:
Make `write_date` behavior the same as v15 for `create_date`, try this (see https://github.com/OCA/helpdesk/blob/77cdcc650300b418cfa88d183713afa6df989024/helpdesk_mgmt/views/helpdesk_ticket_views.xml#L237-L241):
```suggestion
                <field name="write_date" widget="remaining_days" optional="show" />
```

Reply:
Updated and squashed